### DISCUSSION TITLE
installer won't start

### DISCUSSION THREAD

**Discussion by oskarmageek** [Ylianst/MeshCentral#6336] [Issues]
Hello, I have a problem, on one computer I can not install the agent, a window pops up for a second and nothing more happens.

Antivirus does not block, running as admin does not do anything.

https://github.com/user-attachments/assets/1650d43d-2980-4135-a7d8-60c741542a23

**Comment by jurecifilho** [comment]
Hello everyone!

I have the same problem here.

Please note that the version of Win11 I am running is
Windows 11 Pro
Version 24H2
Compilação 26100.1742
Windows Feature Expirence Pack 1000.26100.18.0

Any solution?

**Comment by si458** [comment]
24h2 requires wmi so make sure that's enabled for starters,
Also the issue is a corrupt database in windows which needs repairing on ur PC
Will update link here when u find it

**Reply by si458** [reply]
link here for WMI setup https://github.com/Ylianst/MeshCentral/issues/6162#issuecomment-2168031763

link to repair the WMI database https://github.com/Ylianst/MeshAgent/issues/121#issuecomment-1024822418

**Comment by jurecifilho** [comment]
Perfect!  I followed your information and it worked, activate via cli like this:

" DISM  /Online  /Add-Capability  /CapabilityName:WMIC~~~~ "

Case closed.

Thanks.
